external/lein-repo/src/lein_repo/plugin.clj

```diff
@@ -124,11 +124,13 @@
    :repositories merge
    :plugins concat-distinct
    :repl-options (fn [p1 p2] (merge-with concat-distinct p1 p2))
-   :profiles (fn [p1 p2] (merge-with merge-projects p1 p2))
+   :profiles (fn [p1 p2]
+               (merge-with merge-projects p1 p2))
 
    ;; cljx specific
    :prep-tasks concat-distinct
-   :cljx (fn [a b] {:builds (concat-distinct (:builds a) (:builds b))})})
+   :cljx (fn [a b] {:builds (concat-distinct (:builds a) (:builds b))})
+   :jvm-opts concat-distinct})
```

Comment:
Maybe put this under the `:profiles` line above? As written it looks like the "cljx specific" comment applies to this too.